services/frontend/src/utils/models/Project.js

```diff
@@ -77,7 +77,7 @@ export default class Project {
 
             project.skills = project.skills.reduce((acc, skillId) => {
                 if (skillId in skillsById) {
-                    acc = [...acc, skillsById[skillId].name];
+                    acc = [...acc, {name: skillsById[skillId].name}];
```

Comment:
Instead of converting this to just a plain object, I think it'd better convey intent to instead instantiate `Skill` objects, i.e.

```suggestion
                    acc = [...acc, new Skill({name: skillsById[skillId].name})];
```

This way, the downstream users of these objects can rely on a consistent interface.

Reply:
Made a slightly different change, used:
```
acc = [...acc, skillsById[skillId]];
```
Since skillsById already has our skill objects and if we are going to use new Skill() we are going to have the `createdAt`, `updatedAt` and `Id` properties anyways.

Reply:
This works too!